runtime/doc/options.txt

```diff
@@ -6521,7 +6521,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	actually used.
 	If Vim was compiled with the |+emacs_tags| feature, Emacs-style tag
 	files are also supported.  They are automatically recognized.  The
-	default value becomes "./tags,./TAGS,tags,TAGS", unless case
+	default value becomes "./tags;,./TAGS,tags,TAGS", unless case
 	differences are ignored (MS-Windows).  |emacs-tags|
```

Comment:
Given the default for when `|+emacs_tags|` is available is already mentioned above, I think most of this sub-section can be removed.